Added How-to Guides section (#121)
osrf/ros2multirobotbook: src/howto_guides.md

# Introduction

> How-to Guides provide **direct and modular answers** to “How-to” questions regarding **specific topics or questions** you may have in Open-RMF. 

Reviewer: ```suggestion
How-to Guides provide **direct and modular answers** to “How-to” questions regarding **specific topics** you may have about Open-RMF.
```

Author: Removed bold texts in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/45750d57f4ae3b217295ab5f3f3777fe726f73e3).

Author: Changed phrasing in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/33e3ae18af8dbee2988c15bce082c72546ff7f39).

> They contain **step by step instructions** you can follow to quickly get started with key aspects of Open-RMF which is otherwise not covered in previous sections.

Reviewer: ```suggestion
They contain **step-by-step instructions** you can follow to quickly get started with key aspects of Open-RMF that are not covered in previous sections.
```

Author: Addressed in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/8956532db8090ec65504a6f6a952a428d5b4c8c3).

> If there are any interesting topics you would like to **contribute**, **feel free to open a Pull Request [here](https://github.com/osrf/ros2multirobotbook)**.

Reviewer: Let's remove the bold formatting in this sentence, otherwise LGTM!

Author: Done. Addressed in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/413e0be8fe6a743c0dba329131e2ea641a8ffc54).

> If there are any interesting topics you will like to contribute to the wider Open-RMF community, feel free to write up an article and share it here through a Pull Request.

Reviewer: I missed out on this correction, please update accordingly:

```suggestion
If there are any interesting topics you would like to contribute to the wider Open-RMF community, feel free to write up an article and share it here through a Pull Request.
```

Author: Addressed in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/da24e7155e0ef33b34f836b6d4572298f9401a19).

> If there are any interesting topics you will like to **contribute to the wider Open-RMF community**, **feel free to write up an article and share it here through a Pull Request**.

Reviewer: ```suggestion
If there are any interesting topics you will like to **contribute**, **feel free to open a Pull Request [here](https://github.com/osrf/ros2multirobotbook)**.
```

Author: Addressed in [this commit](https://github.com/osrf/ros2multirobotbook/pull/121/commits/2b93bec0f988faec1694aa44cc2524febe7c82ff).
